benchmark/Benchmarks_Rodas.jl

```diff
@@ -0,0 +1,137 @@
+using DifferentialEquations, Plots, LaTeXStrings, LinearAlgebra
```

Comment:
This should go into SciMLBenchmarks.jl so it can run on the clusters and get automatically updated.